anyway/backend_constants.py

```diff
@@ -120,3 +83,183 @@ class ResolutionCategories(Enum):
 
 
 BE_CONST = BackEndConstants()
+
+
+class LabeledCode(Enum):
+
+    def get_label(self) -> str:
+        return self.labels()[self]
+
+    @classmethod
+    def codes(cls: Iterable) -> List[int]:
+        if isinstance(cls, Iterable):
+            return [a.value for a in cls]
+        else:
+            raise ValueError(f'{cls}: needs to be derived from Enum')
+
+    def labels(self):
+        return {}
+
+
+# This is a type for the field 'injury_severity' in the table 'involved_markers_hebrew'
+class InjurySeverity(LabeledCode):
+    FATAL = 1
+    SEVERE = 2
+    LIGHT = 3
+
+    def labels(self):
+        return {
+            InjurySeverity.FATAL: "fatal",
```

Comment:
how come injury severity = 1 is fatal? it should be killed and הרוג in hebrew (In translated label of course, but in code I assume it make sense to have it that way as well)